client/src/components/Header.css

```diff
@@ -0,0 +1,46 @@
+.Header {
+  display: grid;
+  grid-template-columns: [logo] 60px [navigation] auto;
+  padding: 20px;
```

Comment:
Paddings should use `em` or `rem` (em if you want the padding to depend on the font-size of the element, rem on the root font-size)